src/ucs/datastruct/arbiter.h

```diff
@@ -269,7 +269,7 @@ static inline void ucs_arbiter_group_desched(ucs_arbiter_t *arbiter,
 }
 
 /**
- * @return whether the group does not have any queued elements.
+ * @return whether the element is queued in some of the arbiter groups.
```

Comment:
whether the element is queued in an arbiter group.
(an element can't be queued more than once)